make/helm.mk

```diff
@@ -0,0 +1,60 @@
+##@ Helm Charts
+
+.PHONY: helm-build
+helm-build: $(YQ) kustomize manifests ## Build the helm chart from kustomize manifests
+	# Replace the controller image
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMAGE)
+	# Build the helm chart templates from kustomize manifests
+	$(KUSTOMIZE) build config/helm > charts/authorino-operator/templates/manifests.yaml
+	V="$(BUNDLE_VERSION)" $(YQ) -i e '.version = strenv(V)' charts/authorino-operator/Chart.yaml
+	V="$(CERT_MANAGER_VERSION)" $(YQ) -i e '(.dependencies[] | select(.name == "cert-manager").version) = strenv(V)' -i charts/authorino-operator/Chart.yaml
+	wget -O charts/authorino-operator/crds/cert-manager-manifests.yaml https://github.com/cert-manager/cert-manager/releases/download/v${CERT_MANAGER_VERSION}/cert-manager.yaml
```

Comment:
Can we rename `charts/crds` to something like `charts/dependencies` or so?

cert-manager's manifests include a lot more than CRDs (namespaces, deployments, etc.) Additionally, authorino-operator's manifests (the one under `charts/templates`) also include CRDs, yet it's not in `charts/crds`.

Reply:
It was my fault regarding the set of manifests installed related to cert-manager, now it's installed just the CRDs. 
Regarding the `charts/crds` directory, Helm is using that directory to install any needed CRD before its mechanism of installing charts and its dependencies kicks in. I thought it was a good place to host the cert-manager CRDs that are require by its chart https://artifacthub.io/packages/helm/cert-manager/cert-manager

Reply:
Ok, decided to remove the dependency and add the extra steps, will be documented on the repo as well.